NickvisionTubeConverter.Shared/Models/VideoUrlInfo.cs

```diff
@@ -61,14 +66,26 @@ private VideoUrlInfo(string url)
                         { "merge_output_format", "/" }
                     };
                     Python.Runtime.PyDict videoInfo = ytdlp.YoutubeDL(ytOpt).extract_info(url, download: false);
-                    if (videoInfo.HasKey("playlist_count"))
+                    if (videoInfo.HasKey("entries"))
                     {
-                        videoUrlInfo.IsPlaylist = true;
+                        var entries = videoInfo["entries"].As<Python.Runtime.PyList>();
+                        foreach (var e in entries)
+                        {
+                            var entry = e.As<Python.Runtime.PyDict>();
+                            videoUrlInfo.Videos.Add(new VideoInfo()
+                            {
+                                Url = entry["webpage_url"].As<string>(),
+                                Title = entry.HasKey("title") ? entry["title"].As<string>() ?? "Video" : "Video"
```

Comment:
I don't mind that we are "youtube-oriented" in the first place when thinking about TC and writing the code, but I think we should also keep in mind that yt-dlp is capable of downloading audio files (I'm not talking about extracting audio from video, but about downloading media files that are audio from the beginning - files from bandcamp, for example). And so the default title should be neutral in that regard.
This problem also applies to the app's description.

Reply:
Perhaps, `Unknown` or `Media`?

Reply:
> I'm not talking about extracting audio from video, but about downloading media files that are audio from the beginning - files from bandcamp, for example

We should probably add a check for this and disable selecting video formats as well if this is the case.

> Perhaps, Unknown or Media?

I think `Media` would be good.

> This problem also applies to the app's description.

I mean maybe the description is ok since youtube-dl's description is: "Command-line program to download videos from YouTube.com and other video sites"

Reply:
I like `Media` for single file. For playlist, maybe `Playlist Item` + numbering feature suggested in the issue should be force-enabled, otherwise only one file will be downloaded.